src/containers/CalculatorContainers.js

```diff
@@ -50,6 +50,7 @@ class CalculatorContainers extends Component {
 }
 
 const mapStateToProps = state => {
+    console.log(state);
```

Comment:
console.log is for debugging, shouldn't be in a merge.

Reply:
I accidentally left him, I will remove that.